rclcpp_action/src/types.cpp

```diff
@@ -22,12 +22,18 @@ namespace rclcpp_action
 std::string
 to_string(const GoalUUID & goal_id)
```

Comment:
it would be probably nice to update the doc section in https://github.com/ros2/rclcpp/blob/1fd5a96561166ad2ad557bbc1a297cd652883cb2/rclcpp_action/include/rclcpp_action/types.hpp#L37

that conforms to [RFC 4122](https://www.rfc-editor.org/rfc/rfc4122) UUID string representation?